src/udev/udevadm-trigger.c

```diff
@@ -48,6 +72,23 @@ static int exec_list(
                         continue;
                 }
 
+                /* XOR arg_entry_match and arg_entry_nomatch, because in case both are set or unset
+                 * respectively, we just want to fall through and process all entries.
+                 */
+                if (arg_entry_match != arg_entry_nomatch) {
+                        r = db_present(d);
+
+                        if (!r) {
+                                if (arg_entry_nomatch)
+                                        continue;
+                        } else if (r == ENOENT) {
+                                if (arg_entry_match)
+                                        continue;
+                        } else {
+                                log_warning_errno(r, "Failed to check presence of %s in udev db", syspath);
+                        }
```

Comment:
no {} around single-line if blocks

Reply:
Even if the "if" statement is multi-line?